content/applications/websites/website/configuration/cookies_bar.rst

```diff
@@ -2,75 +2,52 @@
+and stored by your browser and track user information like login details, preferences, and browsing
+history. **Essential cookies** are necessary for the website to function, while **optional cookies**
+analyze behavior or display ads.
 
-- Essential cookies, which are necessary for the website to function properly;
-- Non-essential or optional cookies, which are used to analyze your behavior or display
-  advertisements.
-
-.. image:: cookies_bar/popup.png
-   :width: 500
-   :alt: Example of a cookies bar with the popup layout.
-
-Notifying users about data collection, as well as its methods and purposes, is required by data
-protection laws such as `GDPR <https://gdpr.eu>`_. Cookies bars are commonly used to fulfill this
-obligation in a user-friendly and transparent manner. They are displayed immediately upon a user's
-first visit to inform them that the website uses cookies and let them decide whether they want to
-store non-essential cookies on their device.
+Data protection laws require notifying users about data collection methods and purposes.
+**Cookie bars** fulfill this obligation by informing users on their first visit and allowing them to
+decide whether to store all or only essential cookies on their devices.
 
 Configuration
 =============
 
-To enable the cookies bar on your Odoo website, go to :menuselection:`Website --> Configuration` and
-enable :guilabel:`Cookies Bar` in the :guilabel:`Privacy` section.
+To enable the cookies bar on your website, go to :menuselection:`Website --> Configuration -->
+Settings` and enable :guilabel:`Cookies Bar` in the :guilabel:`Privacy` section.
 
-You can :ref:`customize the appearance of your cookies bar <cookies-bar/customization>` and :ref:`edit the content of
-the related Cookie Policy page <cookies-bar/policy>`.
+.. note::
+   The :ref:`Cookie Policy <cookies-bar/policy>` page (/cookie-policy) is automatically created when
+   you enable the cookies bar.
 
 .. _cookies-bar/customization:
 
 Customization
 =============
 
-To adapt the display of the cookies bar, click :guilabel:`Edit` and select the :guilabel:`Cookies
-Bar` building block in the :guilabel:`Invisible Elements` at the bottom of the panel. Customize it
-using the options in the :guilabel:`Customize` tab in the edit panel. Three :guilabel:`Layouts` are
-available:
-
-- :guilabel:`Discrete`: thin bar
-- :guilabel:`Classic`: banner
-- :guilabel:`Popup`: you can change the popup's :guilabel:`Position` to the :guilabel:`Top`,
-  :guilabel:`Middle`, or :guilabel:`Bottom` of the screen.
+From your website, click :guilabel:`Edit`, go to the :guilabel:`Invisible Elements` section at the
+bottom of the panel, and click :guilabel:`Cookies Bar`. You can select a :guilabel:`Layout`, adjust
+the :guilabel:`Size` of the cookie bar, and enable :guilabel:`Backdrop` to gray out the page in the
+background when the cookies bar is displayed on the screen.
```

Comment:
```suggestion
bottom of the panel, and click :guilabel:`Cookies Bar`. You can modify the :guilabel:`Layout` and :guilabel:`Size` of the cookie bar, and enable :guilabel:`Backdrop` to gray out the page in the
background when the cookies bar is displayed on the screen.
```
- I prefer "modify the layout" since the Layout field is not empty by default (you could say "select another :guilabel:`Layout` to be more exact, but this sounds very bof to me)
- *Modify* is the same as *adjust* in this context, so I suggest removing *adjust* to simplify and shorten the sentence a little bit.

Reply:
I've chosen not to describe each type of layout because, in my opinion, it doesn't add value to the documentation. Users will try out the different layouts and see that they can change the position if they select Pop-up. I aimed to be more concise.